2022/27xxx/CVE-2022-27893.json

```diff
@@ -1,18 +1,106 @@
 {
```

Comment:
{'containers': {'cna': {'affected': [{'defaultStatus': 'unaffected', 'packageName': 'osisoft-pi-web-connector', 'product': 'Foundry', 'vendor': 'Palantir Technologies', 'versions': [{'lessThan': '0.44.0', 'status': 'affected', 'version': '0.15.0', 'versionType': 'custom'}]}], 'datePublic': '2022-11-04T14:17:00.000Z', 'descriptions': [{'lang': 'en', 'supportingMedia': [{'base64': False, 'type': 'text/html', 'value': 'The Foundry Magritte plugin osisoft-pi-web-connector&nbsp;versions 0.15.0 - 0.43.0 was found to be logging in a manner that captured authentication requests. This vulnerability is resolved in osisoft-pi-web-connector version 0.44.0.&nbsp;'}], 'value': 'The Foundry Magritte plugin osisoft-pi-web-connector\xa0versions 0.15.0 - 0.43.0 was found to be logging in a manner that captured authentication requests. This vulnerability is resolved in osisoft-pi-web-connector version 0.44.0.\xa0'}], 'impacts': [{'descriptions': [{'lang': 'en'}]}], 'metrics': [{'cvssV3_1': {'attackComplexity': 'LOW', 'attackVector': 'LOCAL', 'availabilityImpact': 'NONE', 'baseScore': 4.2, 'baseSeverity': 'MEDIUM', 'confidentialityImpact': 'HIGH', 'integrityImpact': 'NONE', 'privilegesRequired': 'HIGH', 'scope': 'UNCHANGED', 'userInteraction': 'REQUIRED', 'vectorString': 'CVSS:3.1/AV:L/AC:L/PR:H/UI:R/S:U/C:H/I:N/A:N', 'version': '3.1'}, 'format': 'CVSS', 'scenarios': [{'lang': 'en', 'value': 'GENERAL'}]}], 'problemTypes': [{'descriptions': [{'cweId': 'CWE-532', 'description': 'CWE-532 Insertion of Sensitive Information into Log File', 'lang': 'en', 'type': 'CWE'}]}], 'providerMetadata': {'orgId': '00000000-0000-4000-9000-000000000000'}, 'references': [{'url': 'https://github.com/palantir/security-bulletins/blob/main/PLTRSEC-2022-03.md'}], 'source': {'discovery': 'UNKNOWN'}, 'title': 'Foundry Magritte plugin osisoft-pi-web-connector was logging credentials', 'x_generator': {'engine': 'Vulnogram 0.1.0-dev'}}}, 'cveMetadata': {'assignerOrgId': '00000000-0000-4000-9000-000000000000', 'cveId': 'CVE-2022-27893', 'requesterUserId': '00000000-0000-4000-9000-000000000000', 'serial': 1, 'state': 'PUBLISHED'}, 'dataType': 'CVE_RECORD', 'dataVersion': '5.0'} is not valid under any of the given schemas